packages/mgt-components/src/components/mgt-file-list/mgt-file-list.ts

```diff
@@ -595,14 +580,12 @@ export class MgtFileList extends MgtTemplatedComponent {
           class="file-list"
           tabindex="0"
           @keydown="${this.onFileListKeyDown}"
-          @keyup="${this.onFileListKeyUp}"
-          @blur="${this.onFileListOut}"
-        >
+          @blur="${this.onFileListOut}">
           ${repeat(
             this.files,
             f => f.id,
             f => html`
-              <li class="file-item" @click=${e => this.handleItemSelect(f, e)}>
+              <li class="file-item" @click=${(e: UIEvent) => this.handleItemSelect(f, e)}>
```

Comment:
I think that we need to set an initial tab index of 0 on the first item to ensure that it can be the row that gets focus

The tab index on other items should be -1 and this should change as we move through the list, that way there is a focus ring on the list item where the keyboard focus is.